src/engine/controls/bpmcontrol.cpp

```diff
@@ -278,20 +279,13 @@ void BpmControl::slotControlBeatSyncPhase(double value) {
 }
 
 void BpmControl::slotControlBeatSyncTempo(double value) {
-    if (value == 0) {
-        return;
-    }
-
-    syncTempo();
+    m_pSyncEnabled->set(value != 0);
```

Comment:
This is the momentary sync of tempo only when right clicking the sync button. 
Mapping this to sync enabled will also sync the phase = seeks the track. 
The original use case is syncing the tempo without seeking. This is now no longer possible.

Reply:
moved these controls to the synccontrol and reimplemented using the correct code.